# This represents an orientation in free space in euler form.

float64 ori_p
float64 ori_r
float64 ori_y
